lib/create-esbuild-script.js

```diff
@@ -3,14 +3,14 @@ export { createEsbuildScript };
 import { writePrettyFile } from "./write-pretty-file.js";
 
 async function createEsbuildScript(answers) {
-  const nodeBuildOptions = `  // Build a CJS Node.js bundle
+  const nodeBuildOptions = `  // Build an ESM Node.js bundle
```

Comment:
I feel we are at a point where we should test the output of running this script is the expected one. Do you think it's worth investing in this?